cfgov/core/lib/PDFreactor.py

```diff
@@ -0,0 +1,2268 @@
+"""This is the PDFreactor 6.3 Python API 
```

Comment:
Okay, it probably deserves an entry in https://github.com/cfpb/cfgov-refresh/blob/flapjack/TERMS.md#exceptions then.